pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/addon/PylonAddon.kt

```diff
@@ -89,6 +89,10 @@ interface PylonAddon : Keyed {
     fun mergeGlobalConfig(from: String, to: String): Config {
         require(from.endsWith(".yml")) { "Config file must be a YAML file" }
         require(to.endsWith(".yml")) { "Config file must be a YAML file" }
+        val cached = globalConfigCache[Pair(from, to)]
```

Comment:
```suggestion
        val cached = globalConfigCache[from to to]
```
ngl this may be confusing without highlighting, feel free to take this comment as optional